images/requirements.md

```diff
@@ -1,10 +0,0 @@
----
-title: Image Requirements
```

Comment:
If you change this, don't you also need to modify the manifest?

Reply:
we moved the contents of this PR to Images > Structure, so this isn't needed anymore.